### DISCUSSION TITLE
How to completely uninstall including overlays?

### DISCUSSION THREAD

**Discussion by cozykitten** [Mahmud0808/Iconify#295] [Q&A] [answered]
How do I completely uninstall Iconify with all modifications it did?

The "Remove Everything" button worked fine with sideloading new OTA's, however before updating to A14 I'd like to get my phone back to stock and uninstalling the app doesn't remove all the installed overlays.

**Comment by Mahmud0808** [comment, marked as answer]
Remove the magisk/ksu module that has been automatically installed.

**Reply by cozykitten** [reply]
Thank you, does this one contain the overlay apps so it works systemlessly?
That might be why OTA went without any issues too, since I disable all modules before I sideload.

**Reply by Mahmud0808** [reply]
Yeah iconify is 100% systemless. So no worries.
